py/confluent/build/download_artifacts/constants.py

```diff
@@ -0,0 +1,15 @@
+from enum import Enum
+
+BASE_URL = "https://nexus.confluent.io/service/rest/v1/search/assets?q="
+M2 = "~/.m2/settings.xml"
+PACKAGE_LITERAL = "-package"
+ARTIFACTID = "&maven.artifactId="
+USERNAME_REGEX = r"<username>(.*)</username>"
+PASSWORD_REGEX = r"<password>(.*)</password>"
+FILENAME_REGEX = r"\d{8}.\d{6}-(\d*)-.*(.zip|.tar.gz)$"
+
+VERSION_REGEX = r"<version>(.*?)(|-SNAPSHOT)</version>"
+NAME_REGEX = r"<name>(.*?)(|-parent)</name>"
+SNAPSHOT_LITERAL = "-SNAPSHOT"
+CONFLUENT_LITERAL = "confluent-"
+POM = "pom.xml"
```

Comment:
Refer this to see how to define enums:
https://github.com/confluentinc/release-tools/blob/30d1139fb626789dfd3cb448ef7879b2fc457343/confluent/release/constants.py#L13

Reply:
Some good info on how to style python using pep8
https://www.python.org/dev/peps/pep-0008/#constants